### ISSUE TITLE
Auto_Search instance has no attribute 'smile_string'

### ISSUE THREAD

**Issue by hectormartinezdev** [curieshicy/JRgui#1]
Platform: Ubuntu 16.04 LTS
RDKIT: 2016.03.1

Hey! When I start the GUI I select Auto-Detect and run with the default SMILES `CC1=CC(=C(C=C1)C(=O)O)O`. Then I press "Save Predicted Results" and when I try to save to a file, the console shows this exception:

```
Exception in Tkinter callback
Traceback (most recent call last):
  File "/usr/lib/python2.7/lib-tk/Tkinter.py", line 1540, in __call__
    return self.func(*args)
  File "jrgui.py", line 958, in saveresults
    for string in self.smile_string:
AttributeError: Auto_Search instance has no attribute 'smile_string'
```
This attribute is defined with the method `Auto_Search.popoutwindow` and is only triggered when loading a file of SMILES. So when I try to save the properties of the default SMILES, I didn't use that button so the attribute is not there.

Hope it helps!

**Comment by curieshicy** [comment]
Hi Hector,
Thanks for reporting this. 
When I designed Auto Mode, I expected users to store their SMILES strings in a txt file, and then use "Or Load Your SMILES File" to read in each SMILES, and click "Save Predicted Results" to save the results. In other words, "Or Load Your SMILES file" and "Save Predicted Results" have to be used together. 
So if you store the default SMILES (CC1=CC(=C(C=C1)C(=O)O)O) into a text file, and load it and save the predicted results, it worked alright. 

"Input your SMILES" is for a quick calculation of certain properties, and is supposed to use with "Compute Physical Properties". 

I wrote these instructions at the header of Auto Mode GUI window.

By the way, from the error message I see you are likely using python 2.7? I believe JRgui requires a Python 3+.

Hope this helps.
Chenyang
